code/game/machinery/doors/airlock.dm

```diff
@@ -1037,6 +1037,8 @@
 			return FALSE
 		use_power(50)
 		playsound(src, doorOpen, 30, TRUE)
+		//TODO SEND_SIGNAL
+		SEND_SIGNAL(src, COMSIG_AIRLOCK_OPEN, )
```

Comment:
```suggestion
		SEND_SIGNAL(src, COMSIG_AIRLOCK_OPEN, forced)
```
Always pass the args of the proc

Reply:
Also, place this signal right after `operating = TRUE`
We want to send it right before the event happens. The condition for a door to begin opening, without a chance to go back, is when that happens.